src/Api/Models/Api/Request/Organizations/ImportOrganizationUsersRequestModel.cs

```diff
@@ -1,10 +1,10 @@
 ﻿using Bit.Core.Models.Business;
-using Bit.Core.Models.Table;
+using Table = Bit.Core.Models.Table;
```

Comment:
Was this aliasing necessary? And if so, what is using it? I would expect some changes to use the new `Table` alias

Reply:
We already were using the prefix before. Line 29. `Group = new Table.Group`.